interdiff.go

```diff
@@ -92,41 +96,49 @@ func InterDiffPath(sourcePath string, oldDiff, newDiff io.Reader) (string, error
 	return "", nil
 }
 
-// MixedModeDiff does things
+// MixedMode computes the diff of a oldSource file patched with oldDiff
+// and the newSource file patched with newDiff.
 func MixedMode(oldSource, newSource io.Reader, oldFileDiff, newFileDiff *diff.FileDiff) (string, error) {
 	oldSourceContent, err := readContent(oldSource)
-
 	if err != nil {
 		return "", fmt.Errorf("reading content of OldSource: %w", err)
 	}
 
 	newSourceContent, err := readContent(newSource)
-
 	if err != nil {
 		return "", fmt.Errorf("reading content of NewSource: %w", err)
 	}
 
 	updatedOldSource, err := ApplyDiff(oldSourceContent, oldFileDiff)
-
 	if err != nil {
 		return "", fmt.Errorf("applying diff to OldSource: %w", err)
 	}
 
 	updatedNewSource, err := ApplyDiff(newSourceContent, newFileDiff)
-
 	if err != nil {
 		return "", fmt.Errorf("applying diff to NewSource: %w", err)
 	}
 
-	// TODO: Get diff from updatedOldSource and updatedNewSource
-
-	// TODO: Convert got diff (chunks) in to FileDiff
+	ch := diffFinder.DiffChunks(strings.Split(strings.TrimSuffix(updatedOldSource, "\n"), "\n"),
+		strings.Split(strings.TrimSuffix(updatedNewSource, "\n"), "\n"))
 
-	// TODO: Manage timestamps
+	// TODO: something with extended (extended header lines)
+	resultFileDiff := diff.FileDiff{OrigName: oldFileDiff.NewName,
+		OrigTime: oldFileDiff.NewTime,
```

Comment:
```suggestion
	resultFileDiff := diff.FileDiff{
		OrigName: oldFileDiff.NewName,
		OrigTime: oldFileDiff.NewTime,
```

Please align members in literal struct initializations.